packages/node_modules/@node-red/nodes/locales/en-US/core/58-debug.html

```diff
@@ -23,4 +23,5 @@ <h3>Details</h3>
        Clicking on the source node id will reveal that node within the workspace.</p>
     <p>The button on the node can be used to enable or disable its output. It is recommended to disable or remove any Debug nodes that are not being used.</p>
     <p>The node can also be configured to send all messages to the runtime log, or to send short (32 characters) to the status text under the debug node.</p>
+    <p>Output of this node can be edited using JSONata expression by specifying the expression on <b>Output</b> field.</p>
```

Comment:
This line of text is no longer matches the latest changes and should be removed.

I would suggest changing the first line of help to:

```
<p>Displays selected message properties in the debug sidebar tab and optionally the runtime log. By default it displays <code>msg.payload</code>, but can be configured to display any property, the full message or the result of a JSONata expression.</p>
```